lib/puppet_x/templates/invoke_ps_command.erb

```diff
@@ -23,7 +215,10 @@ public class PuppetPSHost : PSHost
     public override Guid InstanceId { get { return _hostId; } }
     public override string Name { get { return "PuppetPSHost"; } }
     public override Version Version { get { return new Version(1, 0); } }
```

Comment:
What version would we use? Isn't this the same issue as getting the Host version when over WinRM? or Chocolatey?